app/views/hyrax/dashboard/_collections.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+  <div class="col-md-12">
+    <div id="accordion">
+      <div class="panel panel-default">
+
+        <div class="panel-heading">
+          <h2 class="panel-title"><%= t('.title') %></h2>
+          <div><%= t('.recent_activity') %></div>
+        </div>
+
+        <div class="panel-body">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th><%= t('.collection_title') %></th>
+                <th><%= t('.last_updated') %></th>
+                <th><%= t('.works') %></th>
+                <th><%= t('.files') %></th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+             <% @collections.each do |collection| %>
+                <tr>
+                  <td><%= collection %></td>
+                  <td><%= collection.modified_date.strftime("%Y-%m-%d") %></td>
```

Comment:
i'm worried about these additions, which look like they are computationally complex and likely to be too slow to be tolerable for view code.

especially the implementation of `collection.total_file_sets` (i'll comment further there). if this isn't closely related to the analytics work, i'd suggest we consider it separately so it doesn't hold up other features here.

Reply:
Removed for now (part of pinned collections feature).